src/nl/rubensten/texifyidea/settings/labelDefiningCommands/MyIntFilter.kt

```diff
@@ -0,0 +1,60 @@
+package nl.rubensten.texifyidea.settings.labelDefiningCommands
+
+import java.lang.StringBuilder
+import javax.swing.text.AttributeSet
+import javax.swing.text.DocumentFilter
+
+class MyIntFilter: DocumentFilter() {
+    override fun insertString(fb: FilterBypass?, offset: Int, string: String?, attr: AttributeSet?) {
+        val doc = fb?.document ?: return
+        val content = StringBuilder()
+        content.append(doc.getText(0, doc.length))
+        content.insert(offset, string)
+        if (content.toString().isConvertibleToInt()) {
+            super.insertString(fb, offset, string, attr)
+        }
+        else {
+            TODO("Weiß der Geier was da hin soll...")
```

Comment:
Yup, I noticed them right after pushing and removed them, because you don't really need them.
This class is used to enable only positiv integer input for the position. I added a comment and updated the class name